dataGenerator.js

```diff
@@ -0,0 +1,78 @@
+const faker = require('faker');
+const fs = require('fs');
+
+let movieData = [];
+let fileCount = 1;
+
+let trailerLinks = ['https://www.youtube.com/watch?v=Rp8WL621uGM','https://www.youtube.com/watch?v=WKWFr9upIuU&t=15s','https://www.youtube.com/watch?v=XKB5h2tJQHQ','https://www.youtube.com/watch?v=ZVANQheoRUw','https://www.youtube.com/watch?v=OTd_D7QC8jM&t=17s','https://www.youtube.com/watch?v=WMdBs7uTLHw','https://www.youtube.com/watch?v=adzYW5DZoWs','https://www.youtube.com/watch?v=DltrMStqr8Q','https://www.youtube.com/watch?v=qeau7UWjEms','https://www.youtube.com/watch?v=xRc3WviXk2M','https://www.youtube.com/watch?v=Z6yTXsCauIc','https://www.youtube.com/watch?v=AD10dDdEGmU','https://www.youtube.com/watch?v=mEUsRi74lcY','https://www.youtube.com/watch?v=LiwFtzuSnkE','https://www.youtube.com/watch?v=dFGYK-iRMgw','https://www.youtube.com/watch?v=Hld-7oBn3Rk','https://www.youtube.com/watch?v=6IO8mjyWbhA','https://www.youtube.com/watch?v=i2Ozn_ox_4A','https://www.youtube.com/watch?v=B12REtkl2KY','https://www.youtube.com/watch?v=pU8-7BX9uxs',
```

Comment:
these links are a big unwieldy... would recommend putting them into a separate `json` file that you import into `dataGenerator.js`